Library/Homebrew/formula.rb

```diff
@@ -2628,6 +2643,16 @@ def test(&block)
       define_method(:test, &block)
     end
 
+    # Livecheck can be used to check if there are any updates to the Formula.
+    # WIP
```

Comment:
Would be good to have comments on all these methods (and those in `livecheck.rb`)

Reply:
Sure, I'll get working on them. Just trying to frame them in such a way that they're clear and concise. Looking forward to your suggestions once they're done.

Reply:
Sure thing 👍. They don't need to be more complex than typical methods in `formula.rb`. Think of those in `formula.rb` from the perspective of a contributor trying to write a new formula and in `livecheck.rb` from a Homebrew maintainer wanting to refactor them.